.prettierrc.json

```diff
@@ -7,5 +7,6 @@
   "arrowParens": "avoid",
   "trailingComma": "es5",
   "bracketSameLine": true,
-  "printWidth": 200
+  "printWidth": 200,
+  "endOfLine": "auto"
```

Comment:
Any reason this is `auto` and not `lf`? :)
Having the same eol character for all files would be nice.